### PR TITLE
Add post about the incoming Salta sprint

### PR DESCRIPTION
**Pull request by jmloyola**

### CODE REVIEW

File: _posts/2022-08-09-salta-sprint.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Scikit-learn Sprint in Salta, Argentina"
```
Comment (reshamas) [inline _posts/2022-08-09-salta-sprint.md:2]:
```suggestion
title: "scikit-learn Sprint in Salta, Argentina"
```

File: _posts/2022-08-09-salta-sprint.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  - name: Juan Martín Loyola
+    website: https://jmloyola.github.io/
+    image: juan-martin-loyola.jpeg
+---
+<div>
+  {% include postauthor.html %}
+</div>
+
+In September of 2022, the [SciPy Latin America](https://pythoncientifico.ar/) conference will take place in Salta, Argentina.
+As part of the event, we are organizing a [Scikit-learn Sprint](https://pythoncientifico.ar/events/sprints/) for the people attending.
```
Comment (reshamas) [inline _posts/2022-08-09-salta-sprint.md:21]:
```suggestion
As part of the event, we are organizing a [scikit-learn sprint](https://pythoncientifico.ar/events/sprints/) for the people attending.
```

File: _posts/2022-08-09-salta-sprint.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+The main idea is to introduce the participants to the open source world and help them make their first contribution.
+
+<img src="https://pythoncientifico.ar/static/assets/images/scipy-la-2022_logo.png" alt="SciPy logo">
```
Comment (reshamas) [inline _posts/2022-08-09-salta-sprint.md:24]:
```suggestion
<img src="https://pythoncientifico.ar/static/assets/images/scipy-la-2022_logo.png" alt="SciPy logo" width="50%" height="50%" >
```

File: _posts/2022-08-09-salta-sprint.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    image: juan-martin-loyola.jpeg
+---
+<div>
+  {% include postauthor.html %}
+</div>
+
+In September of 2022, the [SciPy Latin America](https://pythoncientifico.ar/) conference will take place in Salta, Argentina.
+As part of the event, we are organizing a [scikit-learn sprint](https://pythoncientifico.ar/events/sprints/) for the people attending.
+The main idea is to introduce the participants to the open source world and help them make their first contribution.
+
```
Comment (reshamas) [inline _posts/2022-08-09-salta-sprint.md:23]:
```suggestion
The main idea is to introduce the participants to the open source world and help them make their first contribution.  The sprint event is in-person.

```

**Comment by reshamas** [comment]
This looks good!  Thanks @jmloyola.

**Comment by reshamas** [comment]
cc:  @symeneses 
There are a couple of key links in this blog about the upcoming scikit-learn sprint, as part of SciPy Latin America in September 2022.

**Comment by reshamas** [comment]
@jmloyola in your sprint repo (https://github.com/jmloyola/sklearn-sprint-argentina-2022):  
1. Is there info about pair programming (I have only 1 year of Spanish so can read some, but not all Spanish).
2. Should we add an intermediate level issue for participants who may be ready for it? such as this one:
https://github.com/scikit-learn/scikit-learn/issues/23462
3.  If scikit-learn office hours will return in September, we can add a note that folks can follow-up and connect with the scikit-learn team after the sprint event to continue their work.

**Comment by jmloyola** [comment]
> @jmloyola in your sprint repo (https://github.com/jmloyola/sklearn-sprint-argentina-2022):
> 
>     1. Is there info about pair programming (I have only 1 year of Spanish so can read some, but not all Spanish).
> 
>     2. Should we add an intermediate level issue for participants who may be ready for it? such as this one:
>        [Make all estimators use `_validate_params` scikit-learn#23462](https://github.com/scikit-learn/scikit-learn/issues/23462)
> 
>     3. If scikit-learn office hours will return in September, we can add a note that folks can follow-up and connect with the scikit-learn team after the sprint event to continue their work.

1. Yes, there is a paragraph with an overview of the driver-navigator pair-programming technique. There is also a link to [this blog post](https://medium.com/@weblab_tech/pair-programming-guide-a76ca43ff389) explaining more about the topic.
2. Yes :D Though I think most participants will be making their first contributions to open-source, thus the proportion of easy issues should be bigger.
3. Good idea. I can link to the scikit-learn calendar and add some information about the office hours. If it is OK, when we approach the date of the event, I can share the link to join the scikit-learn discord (I think it has a lifespan of five days).

**Comment by reshamas** [comment]
@jmloyola 
cc:  @symeneses 

In your instructions repo, is there information/link about how folks can join the scikit-learn Discord server?

**Comment by jmloyola** [comment]
> In your instructions repo, is there information/link about how folks can join the scikit-learn Discord server?

No yet. I think we have to wait to share the link to join the scikit-learn Discord server. The shared link expires in seven days, thus I'll share it the week before the sprint.

**Pull request merged**